packages/asset-service/src/generateAssetData/ethTokens/extendErc20.ts

```diff
@@ -71,3 +40,90 @@ export const extendErc20 = async (): Promise<TokenAsset[]> => {
     }
   })
 }
+
+export const getIronBankTokens = async (): Promise<TokenAsset[]> => {
+  const ironBankTokens: Token[] = await yearnSdk.ironBank.tokens()
+  return ironBankTokens.map((token: Token) => {
+    return {
+      color: '#FFFFFF',
+      contractType: ContractTypes.ERC20,
+      dataSource: AssetDataSource.YearnFinance,
+      icon: token.icon ?? '',
+      name: token.name,
+      precision: Number(token.decimals),
+      receiveSupport: true,
+      secondaryColor: '#FFFFFF',
+      sendSupport: true,
+      symbol: token.symbol,
+      tokenId: toLower(token.address),
+      caip2: caip2.toCAIP2({
+        chain: ChainTypes.Ethereum,
+        network: NetworkTypes.MAINNET
+      }),
+      caip19: caip19.toCAIP19({
+        chain: ChainTypes.Ethereum,
+        network: NetworkTypes.MAINNET,
+        tokenId: token.address,
+        contractType: ContractTypes.ERC20
+      })
+    }
+  })
+}
+
+export const getZapperTokens = async (): Promise<TokenAsset[]> => {
+  const zapperTokens: Token[] = await yearnSdk.tokens.supported()
+  return zapperTokens.map((token: Token) => {
+    return {
+      color: '#FFFFFF',
```

Comment:
nit pick - we can uses zappers brand color here

Reply:
I'm not sure if these colors are used anywhere (yet). But either way, done.